core/templates/pages/story-editor-page/story-editor-page.component.spec.ts

```diff
@@ -114,8 +114,8 @@ describe('Story editor page', function() {
       thumbnail_filename: null,
       url_fragment: 'story-url-fragment'
     });
-    var MockEditableStoryBackendApiService = {
-      validateExplorations: () => Promise.resolve([])
+    const MockEditableStoryBackendApiService = {
```

Comment:
What you would need to do is update the function in the editable-story-backend-api.service.ts file, add the async keyword and add Async to the end of the end of the name. And then edit all places it is called.